feat(changelog): console-changed-authentication-logs 2025-09-03 (#5494)
scaleway/docs-content: changelog/september2025/2025-09-03-console-changed-authentication-logs.mdx

---
title: Authentication code
status: changed
date: 2025-09-03
category: console
product: console
---

> The login process evolves with the replacement of the magic link by an authentication code sent by email

Reviewer: ```suggestion
The login process has been updated, replacing the magic link with an authentication code sent by email.
```

> The login process has been updated, replacing the magic link with an authentication code sent by email.
> 

Reviewer: ```suggestion
The login process to the Scaleway console has been updated, replacing the magic link with an authentication code sent by email.
```
